src/dataflow/src/render/delta_join.rs

```diff
@@ -110,43 +98,26 @@ where
                                     err_streams.push(errs.leave().leave());
                                 }
 
+                                let mut bound_inputs = vec![relation];
```

Comment:
In reading the PR, this shows up as a new line w/o comments, and I can't tell what it is for. Perhaps tossing in a comment along the lines of "we track the input relations that have been used in a vector for convenience" or something to this effect (if that's correct).